synapse/models/inet.py

```diff
@@ -825,62 +909,235 @@ def getModelDefs(self):
-                        ('port', ('inet:port', {}), {'ro': 1,
-                             'doc': 'The port of the URL. URLs prefixed with http will be set to port 80 and '
-                                 'URLs prefixed with https will be set to port 443 unless otherwise specified.'}),
-                        ('proto', ('str', {'lower': True}), {'ro': 1,
-                             'doc': 'The protocol in the URL.'}),
-                        ('user', ('inet:user', {}), {'ro': 1,
-                             'doc': 'The optional username used to access the URL.'}),
+                    ('inet:web:group', {}, (
+                        ('site', ('inet:fqdn', {}), {
+                            'ro': 1,
+                            'doc': 'The site or service associated with the group.'
+                        }),
+                        ('id', ('inet:group', {}), {
+                            'ro': 1,
+                            'doc': 'The site-specific unique identifier for the group (may be different from '
+                                'the common name or display name).'
+                        }),
+                        ('name', ('inet:group', {}), {
+                            'doc': 'The localized name associated with the group (may be different from '
+                                'the account identifier, e.g., a display name).'
+                        }),
+                        ('name:en', ('inet:group', {}), {
+                            'doc': 'The English version of the name associated with the group (may be different '
+                                'from the localized name).'
+                        }),
+                        ('url', ('inet:url', {}), {
+                            'doc': 'The service provider URL where the group is hosted.'
+                        }),
+                        ('avatar', ('file:bytes', {}), {
+                            'doc': 'The file representing the avatar (e.g., profile picture) for the group.'
+                        }),
+                        ('desc', ('str', {}), {  # FIXME was str:txt
+                            'doc': 'The text of the description of the group.'
+                        }),
+                        ('webpage', ('inet:url', {}), {
+                            'doc': 'A related URL specified by the group (e.g., primary web site, etc.).'
+                        }),
+                        ('loc', ('str', {'lower': True}), {
+                            'doc': 'A self-declared location for the group.'
+                        }),
+                        ('latlong', ('geo:latlong', {}), {
+                            'doc': 'The last known latitude/longitude for the node'
+                        }),
+                        ('signup', ('time', {}), {
+                            'doc': 'The date and time the group was created on the site.'
+                        }),
+                        ('signup:ipv4', ('inet:ipv4', {}), {
```

Comment:
this feels like it rolls up into `inet:client`

Reply:
agreed